domain/src/main/java/com/teamagam/gimelgimel/domain/notifications/SyncMessageNotificationInteractor.java

```diff
@@ -4,31 +4,58 @@
 import com.google.auto.factory.Provided;
 import com.teamagam.gimelgimel.domain.base.executor.PostExecutionThread;
 import com.teamagam.gimelgimel.domain.base.executor.ThreadExecutor;
-import com.teamagam.gimelgimel.domain.base.interactors.SyncInteractor;
-import com.teamagam.gimelgimel.domain.notifications.repository.MessageNotifications;
+import com.teamagam.gimelgimel.domain.base.interactors.BaseSingleDisplayInteractor;
+import com.teamagam.gimelgimel.domain.base.interactors.DisplaySubscriptionRequest;
 import com.teamagam.gimelgimel.domain.notifications.entity.MessageNotification;
-
-import rx.Observable;
-import rx.Subscriber;
+import com.teamagam.gimelgimel.domain.notifications.repository.MessageNotifications;
 
 /**
  * Sync Interactor for Message notifications based on {@link MessageNotification}
  */
 @AutoFactory
-public class SyncMessageNotificationInteractor extends SyncInteractor<MessageNotification>{
+public class SyncMessageNotificationInteractor extends BaseSingleDisplayInteractor {
```

Comment:
The use of "Sync" in the name doesn't suite anymore IMO. DisplayMessageNotificationInteractor ?